### ISSUE TITLE
Versions 1.9.0 and 1.9.1 not accepted by Mac

### ISSUE THREAD

**Issue by VincentIcke** [doxygen/doxygen#8334]
I have Doxygen 1.8.19 installed. Works fine. Downloaded 1.9.0, was not accepted by Mac. 1.9.1 idem. I'll try to attach a screenshot.

specs of my machine:

MacBook Pro mid 2015
2.2 GHz Intel Core i7
macOS 10.14.6

<img width="435" alt="DoxygenFailure" src="https://user-images.githubusercontent.com/76685875/104572911-90b5b080-5654-11eb-96c1-ecb997565b69.png">

**Comment by albert-github** [comment]
I'm not a Mac user, but is there any error message?

**Comment by doxygen** [comment]
@VincentIcke When you launch the DMG file can you send me the output of running this command from the terminal:

```
/usr/bin/otool -L /Volumes/Doxygen/Doxygen.app/Contents/MacOS/Doxywizard 
``` 

I think the problem is the Qt version that requires MacOS version 15.

To check if the command line tool `doxygen` still works on your system please try

```
/Volumes/Doxygen/Doxygen.app/Contents/Resources/doxygen --version
```

**Comment by VincentIcke** [comment]
Hi,

No, there’s no error message. The DMG file shows a grey ‘stop’ traffic sign. Dimitri van Heesch has contacted me with some suggestions, I’ll work on those first. Thanks! - Vincent

**Comment by gsrohde** [comment]
I had this problem as well (with version 1.9.0): double-clicking `Doxygen.app` in the `applications` folder brought up the following message:
> You can’t use this version of the application “Doxygen.app” with this version of macOS.
> You have macOS 10.14.6. The application requires macOS 10.15 or later.
I could use `doxygen` on the command line, but I couldn't use the GUI.

But then I discovered that opening Doxywizard by running `doxywizard` on the command line seemed to work just fine.

**Comment by asmaloney** [comment]
Just ran into this. It looks like a packaging problem starting with the 1.9.0 release.

> A binary distribution for Mac OS X 10.14 and later

Doxygen shouldn't require 10.15 according to the site (and I can't imagine it's using anything that would require 10.15).

>@doxygen:  I think the problem is the Qt version that requires MacOS version 15.

I don't believe there are Qt versions which [require 10.15 yet](https://doc.qt.io/qt-6/macos.html)?

**Comment by albert-github** [comment]
You wrote:
> Just ran into this. It looks like a packaging problem starting with the 1.9.0 release.

- Which versions did you try?
- Did you try the 1.9.2 version as well (released yesterday. August 28, 2021)?

**Comment by asmaloney** [comment]
> Did you try the 1.9.2 version as well

Yes.

> Which versions did you try?

1.9.0, 1.9.1, and 1.9.2.

Fell back to 1.18.20 which works.

**Comment by albert-github** [comment]
No idea where the 10.15 requirement should come from, but I saw a small thing in the Info.plist (in Doxygen/Doxygen.app/Contents):
```
<string>1.9.2, Copyright (c) 1997-2020 by Dimitri van Heesch</string>
```
should probably have to read:
```
<string>1.9.2, Copyright (c) 1997-2021 by Dimitri van Heesch</string>
```
@doxygen something to pay attention to for a next release.

**Comment by doxygen** [comment]
@VincentIcke, @asmaloney, @gsrohde  I think I found a way to build the Doxywizard binary while forcing the deployment target to OSX version 10.14. 


Can you check and let me know if this version works for you: https://www.doxygen.nl/files/Doxygen-1.9.2_OSX.10.14.dmg
My machine is running 10.15, so it is hard for me to test, but `otool` results look promising:

```
$ otool -l /Volumes/Doxygen/Doxygen.app/Contents/MacOS/Doxywizard | grep -A6 LC_BUILD_VERSION
      cmd LC_BUILD_VERSION
  cmdsize 32
 platform 1
    minos 10.14  <------ was 10.15 before
      sdk 11.1
   ntools 1
     tool 3
```

**Comment by asmaloney** [comment]
Yep - that does it! Nice one.

Thanks Dimitri!

**Comment by doxygen** [comment]
@asmaloney Thanks for confirming. I've replaced the original https://www.doxygen.nl/files/Doxygen-1.9.2.dmg with this updated version, and now removed the temporary version to save space.

**Comment by doxygen** [comment]
With the referenced commit it should continue to work for future versions. Since this is now fixed with 1.9.2, I'll close the issue. Please reopen if you still see problems.
